datafusion/sqllogictest/test_files/insert_to_external.slt

```diff
@@ -250,7 +245,6 @@ single_file_test(a bigint, b bigint)
 STORED AS csv
 LOCATION 'test_files/scratch/insert_to_external/single_csv_table.csv'
 OPTIONS(
-create_local_path 'true',
 single_file 'true',
```

Comment:
I intend to replace single_file with inference based on the ListingTableUrl, i.e. if it ends with a path separator, as a follow on PR